docs/conf.py

```diff
@@ -280,76 +275,31 @@
 # 'classic', 'basicstrap'
-html_theme = "basicstrap"
+html_theme = "sphinx_book_theme"
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
 #
 html_theme_options = {
-    # Set the lang attribute of the html tag. Defaults to 'en'
-    "lang": "en",
-    # Disable showing the sidebar. Defaults to 'false'
-    "nosidebar": False,
-    # Show header searchbox. Defaults to false. works only "nosidebar=True",
-    "header_searchbox": False,
-    # Put the sidebar on the right side. Defaults to false.
-    "rightsidebar": False,
-    # Set the width of the sidebar. Defaults to 3
-    "sidebar_span": 3,
-    # Fix navbar to top of screen. Defaults to true
-    "nav_fixed_top": True,
-    # Fix the width of the sidebar. Defaults to false
-    "nav_fixed": True,
-    # Set the width of the sidebar. Defaults to '900px'
-    "nav_width": "300px",
-    # Fix the width of the content area. Defaults to false
-    "content_fixed": False,
-    # Set the width of the content area. Defaults to '900px'
-    "content_width": "900px",
-    # Fix the width of the row. Defaults to false
-    "row_fixed": False,
-    # Disable the responsive design. Defaults to false
-    "noresponsive": False,
-    # Disable the responsive footer relbar. Defaults to false
-    "noresponsiverelbar": False,
-    # Disable flat design. Defaults to false.
-    # Works only "bootstrap_version = 3"
-    "noflatdesign": False,
-    # Enable Google Web Font. Defaults to false
-    "googlewebfont": True,
-    # Set the URL of Google Web Font's CSS.
-    # Defaults to 'http://fonts.googleapis.com/css?family=Text+Me+One'
-    "googlewebfont_url": "http://fonts.googleapis.com/css?family=Roboto+Script+One",  # NOQA
-    # Set the Style of Google Web Font's CSS.
-    # Defaults to "font-family: 'Text Me One', sans-serif;"
-    "googlewebfont_style": "font-family: Helvetica",
-    # Set 'navbar-inverse' attribute to header navbar. Defaults to false.
-    "header_inverse": True,
-    # Set 'navbar-inverse' attribute to relbar navbar. Defaults to false.
-    "relbar_inverse": False,
-    # Enable inner theme by Bootswatch. Defaults to false
-    "inner_theme": False,
-    # Set the name of inner theme. Defaults to 'bootswatch-simplex'
-    # "inner_theme_name": "bootswatch-Yeti",
-    # Select Twitter bootstrap version 2 or 3. Defaults to '3'
-    "bootstrap_version": "3",
-    # Show "theme preview" button in header navbar. Defaults to false.
-    "theme_preview": False,
+    "repository_url": "https://github.com/DeepanshS/mrsimulator",
+    "use_repository_button": True,
+    "use_issues_button": True,
+    "use_download_button": True,
+    "launch_buttons": {
+        "notebook_interface": "jupyterlab",
+        "binderhub_url": "https://mybinder.org",
+        "colab_url": "https://colab.research.google.com",
+    },
```

Comment:
The current example pages built by **Sphinx Gallery** don't have these buttons added. However, the html of the page has a spot where these buttons should go. Adding the launch functionality is a matter of getting **Jupyter Book** to recognize the pages built by **Sphinx Gallery** as needing these buttons.

I think merging this PR is good for now and the issue of adding the buttons can come later.